src/Aspire.Hosting/MySql/PhpMyAdminConfigWriterHook.cs

```diff
@@ -5,7 +5,8 @@
 using Aspire.Hosting.Lifecycle;
 
 namespace Aspire.Hosting.MySql;
-public class PhpMyAdminConfigWriterHook : IDistributedApplicationLifecycleHook
```

Comment:
Does this require a breaking change notice?

Reply:
No. This was added during preview4.